receiver/hostmetricsreceiver/internal/scraper/processscraper/process_scraper_test.go

```diff
@@ -907,21 +922,38 @@ func TestScrapeMetrics_MuteErrorFlags(t *testing.T) {
 			muteProcessNameError: false,
 			muteProcessExeError:  true,
 			muteProcessIOError:   true,
-			expectedError:        fmt.Sprintf("error reading process name for pid 1: %v", processNameError),
+			expectedError: func() string {
+				if runtime.GOOS == "windows" {
+					return fmt.Sprintf("error reading process name for pid 1: %v", processEmptyExeError)
+				}
+				return fmt.Sprintf("error reading process name for pid 1: %v", processNameError)
+			}(),
 		},
 		{
 			name:                 "Process Name Error Enabled And Process Exe Error Enabled And Process IO Error Muted",
 			muteProcessNameError: false,
 			muteProcessExeError:  false,
 			muteProcessIOError:   true,
-			expectedError: fmt.Sprintf("error reading process executable for pid 1: %v; ", processNameError) +
-				fmt.Sprintf("error reading process name for pid 1: %v", processNameError),
+			expectedError: func() string {
+				if runtime.GOOS == "windows" {
+					return fmt.Sprintf("error reading process executable for pid 1: %v; ", processNameError) +
+						fmt.Sprintf("error reading process name for pid 1: %v", processEmptyExeError)
+				}
+				return fmt.Sprintf("error reading process executable for pid 1: %v; ", processNameError) +
+					fmt.Sprintf("error reading process name for pid 1: %v", processNameError)
+			}(),
 		},
 		{
 			name:            "Process Name Error Default (Enabled) And Process Exe Error Default (Enabled) And Process IO Error Default (Enabled)",
 			omitConfigField: true,
-			expectedError: fmt.Sprintf("error reading process executable for pid 1: %v; ", processNameError) +
-				fmt.Sprintf("error reading process name for pid 1: %v", processNameError),
+			expectedError: func() string {
+				if runtime.GOOS == "windows" {
+					return fmt.Sprintf("error reading process executable for pid 1: %v; ", processNameError) +
+						fmt.Sprintf("error reading process name for pid 1: %v", processEmptyExeError)
+				}
+				return fmt.Sprintf("error reading process executable for pid 1: %v; ", processNameError) +
+					fmt.Sprintf("error reading process name for pid 1: %v", processNameError)
+			}(),
```

Comment:
Is there any difference?

Reply:
The last errors are different: `processEmptyExeError`, `processNameError`

Reply:
I see now. thanks